tests/test_parse_fluidx_scan.py

```diff
@@ -0,0 +1,59 @@
+from os.path import join, dirname, abspath
+from unittest.mock import Mock, patch, PropertyMock
+
+import pytest
+
+from EPPs.common import StepEPP
+from scripts.parse_fluidx_scan import ParseFluidXScan
+from tests.test_common import TestEPP, NamedMock
+
+
+class TestParseFluidXScan(TestEPP):
+
+    output1=Mock(id=join(dirname(abspath(__file__)), 'assets',
+                                                      'fluidx_example_scan.csv'))
+    output2=Mock(id=join(dirname(abspath(__file__)), 'assets',
+                                                      'fluidx_example_scan_too_many_samples.csv'))
+
+    container=NamedMock(real_name='container1')
+    location=[container,'A:1']
+    sample1=Mock(udf={'2D Barcode':'-'})
+    input1=Mock(location=location, samples=[sample1])
+
+    patched_process1 = patch.object(
+        StepEPP,
+        'process',
+        new_callable=PropertyMock(return_value=Mock(
+            all_outputs=Mock(return_value=[output1]),
+            all_inputs=Mock(return_value=[input1])
+        )
+    ))
+
+    patched_process2 = patch.object(
+        StepEPP,
+        'process',
+        new_callable=PropertyMock(return_value=Mock(
+            all_outputs=Mock(return_value=[output2]),
+            all_inputs=Mock(return_value=[input1])
```

Comment:
The output on these patch are not being used. Only the file path provided to the command line bellow is.

Reply:
updated